src/fu-util.c

```diff
@@ -1722,6 +1739,8 @@ fu_util_update_by_id (FuUtilPrivate *priv, const gchar *device_id, GError **erro
 	if (!fu_util_update_device_with_release (priv, dev, rel, error))
 		return FALSE;
 
+	/* TRANSLATORS: success message where the specific device is specified */
+	g_print ("%s\n", _("Successfully updated device"));
 	fu_util_display_current_message (priv);
```

Comment:
How about fu_device_get_name here?

Reply:
I think this particular one might need to check if current message is set (like Dell dock will output something).

Maybe even move the output code as part of current message if it's unset?

Reply:
Something like the fixup?

Reply:
Yeah